src/main/scala/io/epiphanous/flinkrunner/flink/state/RichStateUtils.scala

```diff
@@ -0,0 +1,12 @@
+package io.epiphanous.flinkrunner.flink.state
+import scala.collection.JavaConverters._
+
+object RichStateUtils {
+  implicit class RichListState[T](listState: org.apache.flink.api.common.state.ListState[T]) {
+    def _iterator: Iterator[T] = listState.get().iterator().asScala
+    def isEmpty: Boolean = _iterator.isEmpty
+    def contains(element: T): Boolean = _iterator.contains(element)
+    def find(element: T) : T = _iterator.find( v => v.equals(element)).get
```

Comment:
Can we add tests for the `find` and `length` methods?

Reply:
Of course, will do.